addons/crm/tests/test_crm_lead_lost.py

```diff
@@ -121,12 +119,15 @@ def test_lead_lost_crm_rights(self):
                 'name': 'Test Reason'
             })
 
-        lost_wizard = self.env['crm.lead.lost'].with_context({
-            'active_ids': lead.ids
-        }).create({
+        lost_wizard = self.env['crm.lead.lost'].create({
+            'lead_ids': lead.ids,
             'lost_reason_id': lost_reason.id
         })
 
         # nice try little salesman, you cannot invoke a wizard to update other people leads
         with self.assertRaises(AccessError):
+            # because assertRaises clears the cursor, we need to load the leads back in
+            # we use sudo to prevent `_apply_ir_rules` from adding query conditions to prevent
+            # loading 'invalid' leads, as we want them to fail here
+            lost_wizard.sudo().lead_ids
```

Comment:
Mmh a bit strange, can't we create the wizard inside the assertRaises then?

Reply:
Oh yeah, should definitely make more sense. Leaving a short explanation as comment so we don't move it out.